systems/plants/setupLCPmex.cpp

```diff
@@ -0,0 +1,275 @@
+#include "mex.h"
+#include <iostream>
+#include "drakeUtil.h"
+#include "RigidBodyManipulator.h"
+#include "math.h"
+#include <sstream>
+
+using namespace Eigen;
+using namespace std;
+
+template <typename DerivedA, typename DerivedB, typename DerivedC, typename DerivedD> 
+inline void manipulatorDynamics(RigidBodyManipulator *model, MatrixBase<DerivedA> const & q, MatrixBase<DerivedA> const & v, MatrixBase<DerivedB> &H, MatrixBase<DerivedC> &C, MatrixBase<DerivedD> &B)
```

Comment:
maybe give this a different name to avoid confusion with RigidBodyManipulator::manipulatorDynamics.